_posts/2023-04-09-haberdashers-win-sr2023.md

```diff
@@ -0,0 +1,110 @@
+           caption="The 'Greed' arena" %}
+
+## Virtual Competition
+
+The first part of our league this year was a virtual competition, broadcast live from Southampton in March. Across twenty matches, teams competed in a simulated version of the main competition, testing their coding skills against one another. The day was handily won by Haberdashers’ College, coming 1st in all four of their matches, and going on to win the competition as a whole.
+
+## League Matches
+
+At the in person event, teams participated in a further 70 league matches, showcasing a diverse range of robot designs and strategies. The robots displayed impressive agility, precision, and ingenuity, as they navigated the arena to capture tokens and outmanoeuvre their opponents. Team’s had varying designs from suction systems, lifting mechanisms, and even a flywheel designed to launch tokens backwards.
+
+## Knockouts
+
+The knockout rounds saw teams from the league matches pitted against one another in intense, high-stakes battles. With each round, the competition grew fiercer, and the robots were pushed to their limits as they raced to collect tokens and secure their place in the finals.
+
+## The Final
+
+{% include figure.html class="left" src="/images/content/blog/sr2023/final.JPG"
+           caption="Beginning of the SR2023 Final" alt="Three competitors placing their robots into the arena with a crowd of onlookers watching" %}
+
+In the nail-biting final, Haberdashers’ School faced off against Abingdon School, Gymnasium Markt Indersdorf, and Harris Westminster Sixth Form. Harris Westminster Sixth Form managed to successfully grab Haberdashers’ gold token, but were not able to get it solely back to their own scoring zone. Whilst this was happening, Gymnasium Markt Indersdorf tried to grab a gold token with their arms, but missed slightly and instead grabbed the Abingdon School robot, slowly pulling on the robot and some of its wires for the rest of the match! At the same time, the Haberdashers’ robot slowly but steadily collected bronze and silver tokens and deposited them in their zone, before going back and getting more. The teams displayed exceptional skill and teamwork, and after a fierce battle, Haberdashers’ School emerged victorious, claiming the first-place title in Student Robotics 2023.
+
+## Results
+
+- Winner: HAB – Haberdashers’ School
+- 2nd place: ABS – Abingdon School
+- 3rd place: MAI – Gymnasium Markt Indersdorf
+- Committee award: QMC – [Queen Mary’s College](https://www.qmc.ac.uk/). Award sponsored by University of Southampton ECS
+- Rookie award: SHB – [Sherborne School](https://www.sherborne.org/). Award sponsored by ITDev
+- Robot and team image award: HAB – Haberdashers’ School
+- Online presence award: HAM – [Hampton](https://hamptonschool.org.uk/) & [Lady Eleanor Holles](https://www.lehs.org.uk/) Schools
+- Challenges award: QMC – Queen Mary’s College. Award sponsored by Adventurous Machines
```

Comment:
Nit: In the past, we've used a table, which has worked quite well. Perhaps we should copy the one from https://github.com/srobo/team-emails/pull/145/files#diff-b42f5f45d5eebd0d52d4acff01a445f43d15db11d32cdfd8fd51486b2372a9feR14-R23?

I like that we mention sponsors though.

Reply:
Have gone with [0d081e4](https://github.com/srobo/website/pull/498/commits/0d081e4c17e5ed6447275ae8822cb245e269430e) as think a table would look sparse with a sponsor column.